portfolio/src/main/webapp/index.html

```diff
@@ -5,6 +5,11 @@
     <title>Kathleen's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDLKaklumtsGRcF_nsuddtExH1rbMh_G9U&callback=initMap&libraries=&v=weekly"
```

Comment:
We should avoid putting the API key in github.

Use git-secrets (recommended) or a .gitignore file to exclude the file containing the service account credentials.(i.e. including /path/to/api_key.json in the .gitignore file)
Place the file with the API key outside of the Github project and retrieve it by creating an environment variable (i.e. GCP_API_KEY=/path/to/api_key.json).

Reply:
You can see how I implemented this in my maps pull request here, by adding the config.json file https://github.com/lilian-hunt/step/pull/4/files

Reply:
Thank you very much for both of your advice! Yes, I do agree, hiding API key locally securely is very important!
I have made a separate PR #8 to fix this problem.